src/NLog.Targets.Seq/Layouts/CompactJsonLayout.cs

```diff
@@ -16,17 +16,19 @@
 using NLog.Config;
 using NLog.Layouts;
 
-namespace NLog.Targets.Seq
+namespace NLog.Targets.Seq.Layouts
 {
     [ThreadAgnostic]
     class CompactJsonLayout : JsonLayout
     {
         readonly JsonAttribute
-            _timestampAttribute = new JsonAttribute("@t", new SimpleLayout("${date:format=o}")),
-            _levelAttribute = new JsonAttribute("@l", new SimpleLayout("${level}")),
-            _exceptionAttribute = new JsonAttribute("@x", new SimpleLayout("${exception:format=toString}")),
-            _messageAttribute = new JsonAttribute("@m", new FormattedMessageLayout()),
-            _messageTemplateAttribute = new JsonAttribute("@mt", new SimpleLayout("${onhasproperties:${message:raw=true}}"));
+            _timestampAttribute = new("@t", new SimpleLayout("${date:format=o}")),
+            _levelAttribute = new("@l", new SimpleLayout("${level}")),
+            _exceptionAttribute = new("@x", new SimpleLayout("${exception:format=toString}")),
+            _messageAttribute = new("@m", new FormattedMessageLayout()),
+            _messageTemplateAttribute = new("@mt", new SimpleLayout("${onhasproperties:${message:raw=true}}")),
+            _traceIdAttribute = new("@tr", new TraceIdLayout()),
+            _spanIdAttribute = new("@sp", new SpanIdLayout());
```

Comment:
Equivalents of these are available in `NLog.DiagnosticSource`, but it seems better to avoid the dependency.